utils/wasi/build-toolchain

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
Is there a reason for this script?  I think that we have far too many scripts that are adding more complexity to the build.  If it’s too difficult to build with the parameters, that should be exposed: the build system needs to be cleaned up, not patched over.

Reply:
I created this file as same as `utils/android/build-toolchain`, but I agree that it adds more complexity to build system. I removed this file and set some of those options as default option for wasi target.